src/Common.props

```diff
@@ -10,7 +10,7 @@
     <OSPlatform>OS_UNKNOWN</OSPlatform>
     <OSArchitecture>ARCH_UNKNOWN</OSArchitecture>
     <DebugConstant></DebugConstant>
-    <VssApiVersion>0.5.151-private</VssApiVersion>
+    <VssApiVersion>0.5.153-private</VssApiVersion>
```

Comment:
Guessing this was the API change to include getting the isreadyonly vars?

Reply:
Yes. This is the API update the makes the change to the container contract.